rcl/test/rcl/test_get_actual_qos.cpp

```diff
@@ -199,24 +201,24 @@ get_parameters()
 
 #ifdef RMW_IMPLEMENTATION_STR
   std::string rmw_implementation_str = RMW_IMPLEMENTATION_STR;
-  if (!rmw_implementation_str.compare("rmw_fastrtps_cpp") ||
-    !rmw_implementation_str.compare("rmw_fastrtps_dynamic_cpp"))
+  if (rmw_implementation_str == "rmw_fastrtps_cpp" ||
+    rmw_implementation_str == "rmw_fastrtps_dynamic_cpp")
   {
     rmw_qos_profile_t expected_system_default_qos = expected_fastrtps_default_qos_profile();
     parameters.push_back({
       rmw_qos_profile_system_default,
       expected_system_default_qos,
       "publisher_system_default_qos"});
   } else {
-    if (!rmw_implementation_str.compare("rmw_opensplice_cpp") ||
-      !rmw_implementation_str.compare("rmw_connext_cpp") ||
-      !rmw_implementation_str.compare("rmw_connext_dynamic_cpp"))
+    if (rmw_implementation_str == "rmw_connext_cpp" ||
+      rmw_implementation_str == "rmw_connext_dynamic_cpp" ||
+      rmw_implementation_str == "rmw_opensplice_cpp")
     {
       rmw_qos_profile_t expected_system_default_qos = expected_system_default_qos_profile();
       parameters.push_back({
-        rmw_qos_profile_system_default,
-        expected_system_default_qos,
-        "publisher_system_default_qos"});
+       rmw_qos_profile_system_default,
+       expected_system_default_qos,
+       "publisher_system_default_qos"});
```

Comment:
Why did you change this to 1 space of indention?

Actually why was this file changed at all?

Reply:
rmw_qos_profile_t has changed, and this file manually creates them using item initializer.
See: https://github.com/ros2/rmw/blob/ba0a71707236939c08ece8265a18b19758dd3121/rmw/include/rmw/types.h#L211

Since this file was changed for this reason, I've improved the readability of the if statements as
Clang tidy will recommend "if an equality or inequality check can suffice, that is recommended."

The 1 space indentation may have been an accident, ament_uncrustify --refactor was run on this file, it may have been my code style settings that have changed this.

Reply:
Ok, let's fix the indentation, it should always be a multiple of 2 for C++.

I'll re-review the file with this new info in mind, thanks.

Reply:
Ok, the style changes are unrelated to the pr, but ok by me. However, I don't follow what you mean about the initializer list changing. Nothing changed as far as I can tell. The only thing I see in the diff is whitespace changes.

Reply:
The initializer list changes were originally part of this PR, but ended up being merged in through https://github.com/ros2/rcl/pull/416 instead. Sorry for the confusion.

Anyway, will fix the indentation issue, and will keep the use of `==` as it expresses the intent better.